client-web/test/exlibris-app/exlibris-book-tile_test.html

```diff
@@ -0,0 +1,57 @@
+<!doctype html>
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport"
+          content="width=device-width, minimum-scale=1, initial-scale=1, user-scalable=yes">
+
+    <title>exlibris-book-tile test</title>
+
+    <script src="./../../bower_components/webcomponentsjs/webcomponents-loader.js"></script>
+    <script src="./../../bower_components/web-component-tester/browser.js"></script>
+    <script src="./test_variables.js"></script>
+
+    <link rel="import" href="../../src/exlibris-app/exlibris-book-tile.html">
+</head>
+<body>
+
+<test-fixture id="BasicTestFixture">
+    <template>
+        <exlibris-book-tile></exlibris-book-tile>
+    </template>
+</test-fixture>
+
+<script>
+    suite('exlibris-book-tile', function () {
+
+        const BookGridSelector = {
+            STATUS_MESSAGE_SELECTOR: "#status-text",
+            TITLE_SELECTOR: "#title",
+            AUTHOR_SELECTOR: "#author",
+            IRON_IMAGE_SELECTOR: "iron-image"
+        };
+
+        let bookGrid;
+        setup(() => {
+            bookGrid = fixture('BasicTestFixture');
+        });
+
+        test('should display `book` from property', function (done) {
```

Comment:
Should display book details from `book` property? The `book` property name can be changed during development so I wouldn't use it in test name.